### PR TITLE
[action] Add the possibility to update an app to the onesignal action

### LINKED ISSUE #14381: [onesignal] Updating an OneSignal app (PUT request)
#### Motivation Behind Feature
Push notification certificates expire after a certain time (one year). The onesignal action can only create new apps via OneSignal API, but not update existing ones.

#### Feature Description
It would be nice to be able to update an existing OneSignal app (created by the onesignal action, which is using a POST request to the Onesignal API) with a newly created .p12 by using the PUT request on the OneSignal API with the same params. 
https://documentation.onesignal.com/reference#update-an-app

This might be even possible by just adding a new param / flag identifying whether the action should create or update an app.

#### Alternatives or Workarounds
Creating the certificate on the command line, when the old one is expired and manually editing the OneSignal App via browser interface (or via curl).

Thank you.

### PR DESCRIPTION
**Pull request by revolter**
Closes #14381.

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/onesignal.rb [hunk 1]
```diff
@@ -33,60 +44,69 @@ def self.run(params)
...
-        "You can use this action to automatically create a OneSignal application. You can also upload a `.p12` with password, a GCM key, or both."
+        "You can use this action to automatically create or update a OneSignal application. You can also upload a `.p12` with password, a GCM key, or both."
       end
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :app_id,
+                                       env_name: "ONE_SIGNAL_APP_ID",
+                                       sensitive: true,
+                                       description: "OneSignal App ID",
```
Comment (janpio) [inline fastlane/lib/fastlane/actions/onesignal.rb]: maybe add something here that it is to be used to update an app, and to `app_name` that it is meant to create a new one.

**Review by janpio — APPROVED** [review]
Looks good!

As always, tests would be super nice but not a hard requirement for getting this merged of course.

File: fastlane/lib/fastlane/actions/onesignal.rb (hunk 1 above)
Comment (revolter) [reply fastlane/lib/fastlane/actions/onesignal.rb]: Are the new texts clear enough?

**Comment by revolter** [comment]
I'll add some tests

**Review by joshdholtz — APPROVED** [review]
Woooooo, this looks great! I've actually been needing this and thanks for adding tests ❤️

**Pull request merged**